WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.h

```diff
@@ -27,9 +27,10 @@ NS_ASSUME_NONNULL_BEGIN
 - (BOOL)fb_deactivateWithDuration:(NSTimeInterval)duration error:(NSError **)error;
 
 /**
- Return application elements tree in form of nested dictionaries
+  @Param excludedAttributes which needs to be excluded from the response possible values contain frame, enabled, visible, accessible, focused
```

Comment:
please follow the same docstring format as in other methods

Reply:
please describe how the method behaves if nil is passed as the value

Reply:
there's already a `nil` check when this variable is used further. 

`if (excludedAttributes == nil || ![excludedAttributes containsObject:key]) {`

in that case, all the attributes will be included in the response.

Reply:
changed docstring format